src/main/java/seedu/address/logic/commands/viewcommand/ViewTeamCommand.java

```diff
@@ -0,0 +1,25 @@
+package seedu.address.logic.commands.viewcommand;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.model.Model;
+
+public class ViewTeamCommand extends ViewCommand {
```

Comment:
I think apart from Team (and maybe issue) we won't need the view command for the other classes because there isnt additional detail to see about those classes. Correct me if I am wrong though and there is a need for these commands.

Reply:
I think we should keep it, reason being that this is an implementation detail and `Controller` should not care about whether the model has detailed fields or not. So if the model returns the same value for detailed and minimal, it is fine. Also gives is more leeeway to make changes to model w/o breaking Controller

Reply:
I was thinking that e.g. for Participant, when we list we'll only show their names, and when we view a Participant, we'll show all their fields, like email, phone#, etc.